tests/foreman/ui/test_repository.py

```diff
@@ -1039,6 +1042,64 @@ def test_positive_remove_credentials(session, function_product, function_org, fu
         assert not repo_values['repo_content']['upstream_authorization']
 
 
+@pytest.mark.tier2
+@pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
+def test_sync_status_persists_after_task_delete(session, module_prod, module_org, target_sat):
+    """Red Hat repositories displayed correctly on Sync Status page.
+
+    :id: 29b79d15-9b92-4b6e-a1e4-9bf79de99c9b
+
+    :BZ: 1924625
+
+    :customerscenario: true
+
+    :Steps:
+        1. Sync a custom Repo.
+        2. Navigate to Content > Sync Status. Assert status is Synced.
+        3. Use foreman-rake console to delete the Sync task.
+        4. Navigate to Content > Sync Status. Assert status is still Synced.
+
+    :expectedresults: Sync Repositories should be shown as Synced on Sync Status page even
```

Comment:
expectedresults needs to be a list if its going to span multiple lines, this is failing testimony validation.

Reply:
oh, ok